users/api/http/transport.go

```diff
@@ -122,4 +123,9 @@ func encodeError(_ context.Context, err error, w http.ResponseWriter) {
 			w.WriteHeader(http.StatusInternalServerError)
 		}
 	}
+
+	if err != nil {
+		w.Header().Set("Content-Type", contentType)
+		json.NewEncoder(w).Encode(errorRes{Err: err.Error()})
```

Comment:
Not all should be sent to client, only our description.

Also, I do not see the print on our logger (where everything should be printed).

Reply:
Logger already print error so no need to change anything in logger. Errors printed with logger will now be more descriptive because they will have multiple wrapped layers and all of them will be printed in log